src/components/pages/History/PoolShares.tsx

```diff
@@ -1,86 +1,103 @@
-import { useMetadata, useOcean } from '@oceanprotocol/react'
+import { useOcean } from '@oceanprotocol/react'
 import React, { ReactElement, useEffect, useState } from 'react'
 import Table from '../../atoms/Table'
-import { DDO, Logger, MetadataCache } from '@oceanprotocol/lib'
 import PriceUnit from '../../atoms/Price/PriceUnit'
 import Conversion from '../../atoms/Price/Conversion'
 import styles from './PoolShares.module.css'
 import AssetTitle from '../../molecules/AssetListTitle'
+import { gql, useQuery } from '@apollo/client'
+import {
+  PoolShares as PoolSharesList,
+  PoolShares_poolShares as PoolShare
+} from '../../../@types/apollo/PoolShares'
+import web3 from 'web3'
 
-interface Asset {
-  ddo: DDO
-  shares: string
-}
-
-function TotalLiquidity({ ddo }: { ddo: DDO }): ReactElement {
-  const { price } = useMetadata(ddo)
-  const totalLiquidityInOcean = price?.ocean + price?.datatoken * price?.value
+const poolSharesQuery = gql`
+  query PoolShares($user: String) {
+    poolShares(
+      orderBy: balance
+      orderDirection: desc
+      where: { userAddress: $user }
+      first: 1000
+    ) {
+      id
+      balance
+      userAddress {
+        id
+      }
+      poolId {
+        id
+        datatokenAddress
+        lockedValue
+      }
+    }
+  }
+`
 
+function UserLiquidity({ lockedValue }: { lockedValue: string }): ReactElement {
   return (
-    <Conversion
-      price={`${totalLiquidityInOcean}`}
-      className={styles.totalLiquidity}
-    />
+    <Conversion price={`${lockedValue}`} className={styles.totalLiquidity} />
   )
 }
 
+function Symbol({ tokenAddress }: { tokenAddress: string }) {
```

Comment:
we might not need this, how about just getting the symbol within the GraphQL query, [like so](https://subgraph.rinkeby.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph/graphql?query=%7B%0A%20%20poolShares(orderBy%3Abalance%2CorderDirection%3A%20desc%2C%20first%3A%201000%2C%20where%3A%20%7BuserAddress%3A%20%220x4d156a2ef69ffddc55838176c6712c90f60a2285%22%7D)%20%7B%0A%20%20%20%20id%0A%20%20%20%20balance%0A%20%20%20%20userAddress%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%20%20poolId%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20symbol%0A%20%20%20%20%20%20datatokenReserve%0A%20%20%20%20%20%20oceanReserve%0A%20%20%20%20%20%20lockedValue%0A%20%20%20%20%20%20tokens%20%7B%0A%20%20%20%20%20%20%20%20tokenId%20%7B%0A%20%20%20%20%20%20%20%20%20%20symbol%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)?

The `null` for where `OCEAN` should be is a bug though https://github.com/oceanprotocol/ocean-subgraph/issues/13

Reply:
yep, good idea, I tried to find the symbol in the graph but it wasn't looking that deep. About the problem with the null, using the graph there are always 2 tokens returned and one of them has a `null` `tokenId`

Reply:
found that too and reported here oceanprotocol/ocean-subgraph#13 but not blocking cause we work around it in here